examples/06-verif-manual/vm-006-pinched_cylinder.py

```diff
@@ -0,0 +1,520 @@
+ - A one-eighth symmetry model is used. One-fourth of the load is applied
+   due to symmetry.
+
+"""
+# sphinx_gallery_thumbnail_path = '_static/vm6_setup.png'
+
+###############################################################################
+# Start MAPDL
+# ~~~~~~~~~~~
+
+from ansys.mapdl.core import launch_mapdl
+
+# Start mapdl and clear it.
+mapdl = launch_mapdl()
+mapdl.clear()
+
+# Enter verification example mode and the pre-processing routine.
+mapdl.verify()
+mapdl.prep7()
+
+
+###############################################################################
+# Define Element Type
+# ~~~~~~~~~~~~~~~~~~~
+# Set up the element type (a shell-type).
+
+# Type of analysis: Static.
+mapdl.antype("STATIC")
+
+# Define element number.
+elem_num = 1
+
+# Element type: SHELL181.
+mapdl.et(elem_num, "SHELL181")
+
+# Special Features are defined by keyoptions of shell element:
+
+# KEYOPT(3)
+# Integration option:
+# Full integration with incompatible modes.
+mapdl.keyopt(elem_num, 3, 2)  # Cubic shape function
+
+
+###############################################################################
+# Define Material
+# ~~~~~~~~~~~~~~~
+# Set up the material.
+
+# Define material number.
+mat_num = 1
+
+# Define material properties.
+mapdl.mp("EX", mat_num, 10.5E6)
+mapdl.mp("NUXY", mat_num, 0.3125)
+print(mapdl.mplist())
+
+
+###############################################################################
+# Define Section
+# ~~~~~~~~~~~~~~
+# Set up the cross-section properties for a shell element.
+
+# Parametrization of the shell thickness.
+t = 0.094
+
+# Define cross-section number.
+sec_num = 1
+
+# Define shell cross-section.
+mapdl.sectype(secid=sec_num, type_="SHELL", name="shell181")
+mapdl.secdata(t, mat_num, 0, 5)
+
+
+###############################################################################
+# Define Geometry
+# ~~~~~~~~~~~~~~~
+# Set up the keypoints and create the area through the keypoints.
+
+# Define global cylindrical coordinate system.
+mapdl.csys(1)
+
+# Define keypoints by coordinates.
+mapdl.k(1, 4.953)
+mapdl.k(2, 4.953, "", 5.175)
+
+# Generates additional keypoints from a pattern of keypoints.
+mapdl.kgen(2, 1, 2, 1, "", 90)
+
+# Create an area through keypoints.
+mapdl.a(1, 2, 4, 3)
+
+# Define the number of the keypoint where F is applied using inline function.
+q = mapdl.queries
+top_keypoint = q.kp(4.953, 90, 0)
+
+# Plot the lines.
+mapdl.lplot(color_lines=True, cpos='iso')
+
+# Plot the area using PyVista properties.
+mapdl.aplot(title="Display the selected area",
+            cpos="iso",
+            vtk=True,
+            color="#06C2AC",
+            show_line_numbering=True,
+            show_area_numbering=True,
+            show_lines=True)
+
+
+###############################################################################
+# Meshing
+# ~~~~~~~
+# Define line division of the lines, then mesh the area with shell elements.
+
+# Specify the default number of line divisions.
+mapdl.esize("", 8)
```

Comment:
Use kwargs rather than providing empty args. It will seem strange for average Python users to see empty args (passed as empty strings) rather than keyword arguments.
```suggestion
mapdl.esize(ndiv=8)
```